File: frontend/src/types/project.ts
export interface Project {
  id: string
  name: string
  description: string
  created_at: string
  updated_at: string
}

export interface Collection {
  id: string
  project_id: string
  parent_id?: string
  name: string
  sort_order: number
  created_at: string
  updated_at: string
}

File: models/project.go
package models

type Project struct {
	ID          int64  `json:"id"`
	Name        string `json:"name"`
	Description string `json:"description"`
	CreatedAt   string `json:"created_at"`
	UpdatedAt   string `json:"updated_at"`
}

type ProjectStats struct {
	RequestCount    int `json:"request_count"`
	CollectionCount int `json:"collection_count"`
}
